web/src/test/manual/web/issue3701/index.html

```diff
@@ -54,16 +54,16 @@
       });
 
       function togglePredictions() {
-        let newVal = !keyman.modelManager.mayPredict;
-        keyman.osk.banner.setOptions({'mayPredict': newVal});
+        let newVal = !keyman.core.languageProcessor.mayPredict;
+        keyman.core.languageProcessor.mayPredict = newVal;
```

Comment:
Is this a public API? Should it be?

Reply:
I think it'd be wise to workshop a better API for that.  It's what we have for now, though.

Reply:
Agreed. For a future version then.